.github/workflows/examples-esp32.yaml

```diff
@@ -64,6 +64,7 @@ jobs:
                   path: |
                       .environment/gn_out/.ninja_log
                       .environment/pigweed-venv/*.log
+
             - name: Build some M5Stack variations
               timeout-minutes: 60
               run: |
```

Comment:
This runs build_examples.py, which probably does not use CCACHE. could we also update build_examples? I am unsure how - ideally ccache should be default/autodetected and not an explicit flag.